### PR TITLE
dx-1214 added metadata in card, invoice and other features

### PR DESCRIPTION
**Pull request by marinafrydenb**

### CODE REVIEW

File: _includes/payment-order-other-features.md [hunk 1]
```diff
@@ -848,7 +848,13 @@ Content-Type: application/json
         "vatPercent": 2500,
         "amount": 1000,
         "vatAmount": 250
-      }
+      },
+    "metadata": {
```
Comment (asbjornu) [inline _includes/payment-order-other-features.md:852]: The indentation here seems a bit off. Please fix.

File: _includes/recur.md [hunk 2]
```diff
@@ -60,6 +66,7 @@ Content-Type: application/json
...
 | {% icon check %} | └─➔&nbsp;`orderReference`      | `String(50)` | The order reference should reflect the order reference found in the merchant's systems.                                                                                                                                                                                               |
 | {% icon check %} | └─➔&nbsp;`subsite`             | `String(40)` | The subsite field can be used to perform [split settlement][split-settlement] on the payment. The subsites must be resolved with Swedbank Pay [reconciliation][settlement-and-reconciliation] before being used.                                                                      |
+|                  | └➔&nbsp;`metadata`             | `object`      | The keys and values that should be associated with the payment. Can be additional identifiers and data you want to associate with the payment.                                                                                                                                                 |
```
Comment (asbjornu) [inline _includes/recur.md:69]: Should we have a section in "other features" explaining `metadata` in more detail, perhaps?

**Review by asbjornu — CHANGES_REQUESTED** [review]

File: _includes/recur.md (hunk 2 above)
Comment (marinafrydenb) [reply _includes/recur.md:69]: We already have one in all payments and checkout + payment menu :)

Comment (asbjornu) [reply _includes/recur.md:69]: Then let's link to it from the field description?

**Review by arebra — APPROVED** [review]
With the fixes @asbjornu suggested, this will be marvelous!

File: _includes/recur.md (hunk 2 above)
Comment (marinafrydenb) [reply _includes/recur.md]: fixed :)

**Review by asbjornu — APPROVED** [review]
🥇

**Pull request merged**